bylaws.md

```diff
@@ -90,11 +117,11 @@ Foreningens bestyrelsen kan fastsætte regler der lader foreningens medlemmer ud
 
 <!-- [en] -->
 # Dues, finance og sponsorater
-Foreningens medlemmer betaler kontigent til ACM og der opkræves intet selvstændigt kontigent til foreningen.
+The Chapters fiscal year begins on July 1st and closes June 30th.
 
-Foreningen kan ikke modtage eller udbetale penge og fører derfor ikke regnskab.
+The Chapters does not collect membership dues and aims to have no monetary transactions.
 
-Foreningen kan modtage sponsorater i form af eksempelvis lån af lokaler, udstyr, domæner eller tryk af materiale.
+The Chapter can receive sponsored items that are directly related to The Chapters operation such as, but not limited to, equipment, domains or printing of reports.
```

Comment:
Receiving sponsorships and gifts kind of requires the organisation to keep financial records so it can be accountable to its members and the world at large. If publication in the name of the chapter is a central thing it must be clear that there are no outside dependencies or influences. I think keeping financial records are key to that.